src/app/effects/book.ts

```diff
@@ -17,10 +17,13 @@ import * as book from '../actions/book';
 
 /**
  * Effects offer a way to isolate and easily test side-effects within your
- * application. StateUpdates is an observable of the latest state and
- * dispatched action. The `toPayload` helper function returns just
+ * application. 
+ * Here `.map((action: book.SearchAction) => action.payload)` could be
```

Comment:
Go ahead and update the example to use the `toPayload` from `@ngrx/effects` and remove the comment about what its doing.